linux_os/guide/system/permissions/restrictions/poisoning/grub2_page_poison_argument/rule.yml

```diff
@@ -23,7 +23,6 @@ severity: medium
 identifiers:
     cce@rhel7: CCE-82158-7
     cce@rhel8: CCE-80944-2
-    cce@rhcos4: CCE-82673-5
```

Comment:
`CCE-82673-5` went from rule `grub2_page_poison_argument` to `coreos_page_poison_kernel_argument`.
The configuration checked/remediated is almost the same, the difference is that `coreos_page_poison_kernel_argument` checks only the last boot entry.

I'm not sure if this CEE migration is okay, maybe @redhatrises has thoughts on it.

Reply:
I am okay with this until we actually deliver something. Ultimately, I think that creating duplicate rules for something that is  subtly unique is not a great. This is something to figure out in a separate PR.